src/EnergyPlus/AirflowNetwork/src/Solver.cpp

```diff
@@ -2203,6 +2207,48 @@ namespace AirflowNetwork {
                 ShowContinueError(m_state, "..Default value \"SkylineLU\" will be used.");
             }
 
+            // Get inputs for duct sizing
+            AirflowNetworkSimu.AFNDuctAutoSize = false;
+            if (NumAlphas == 10) {
+                if (!UtilityRoutines::SameString(Alphas(10), "None")) {
+                    if (UtilityRoutines::SameString(Alphas(10), "MaximumVelocity")) {
+                        AirflowNetworkSimu.ductSizeMethod = AirflowNetworkSimuProp::DuctSizeMethod::MaxVelocity;
+                    } else if (UtilityRoutines::SameString(Alphas(10), "PressureLoss")) {
+                        AirflowNetworkSimu.ductSizeMethod = AirflowNetworkSimuProp::DuctSizeMethod::PressureLoss;
+                    } else if (UtilityRoutines::SameString(Alphas(10), "PressureLossWithMaximumVelocity")) {
+                        AirflowNetworkSimu.ductSizeMethod = AirflowNetworkSimuProp::DuctSizeMethod::VelocityAndLoss;
+                    } else {
+                        ShowSevereError(m_state,
```

Comment:
Use `format` for the entire string, not to convert `RoutineName` into a string and then concatenate with other strings. With `format` you will do one allocation here instead of seven. I understand that this is an error and its performance is "not important", but it is important to get into the habit of not using low-performance code idioms where higher-performance ones are readily available.